third_party/README.md

```diff
@@ -0,0 +1,8 @@
+# third_party
+This directory copies("vendors") files in the following repositories, as required for compiling proto files.
```

Comment:
Should be "from the following repositories" instead of "in the following repositories"